pkg/syncthing/syncthing.go

```diff
@@ -833,54 +832,17 @@ func (s *Syncthing) HardTerminate() error {
 			continue
 		}
 
-		oktetoLog.Infof("terminating syncthing %d with wait: %s", p.Pid, s.Home)
-		parent, err := getParent(p)
-		if err != nil {
-			oktetoLog.Info("can not find parent")
-		}
-
 		oktetoLog.Infof("terminating syncthing %d with wait: %s", p.Pid, s.Home)
 		if err := terminate(p, true); err != nil {
 			oktetoLog.Infof("error terminating syncthing %d with wait: %s", p.Pid, err.Error())
 		}
-		if parent != nil && parent.Pid != int32(pid) {
-			oktetoLog.Infof("pid: %s / name: %s / cmd: %s", p.Pid, p.Name, p.Cmdline)
-			if err := terminate(parent, true); err != nil {
-				oktetoLog.Infof("error terminating syncthing %d with wait: %s", p.Pid, err.Error())
-				continue
-			}
-		}
+
```

Comment:
This code is moved to wait until the okteto notices that there is another process running